test/integration/cloud_variables_sb2.js

```diff
@@ -0,0 +1,152 @@
+const path = require('path');
```

Comment:
This file and the analogous sb3 file are the same tests just running on sb2s vs. sb3s.